src/core_atmosphere/Registry.xml

```diff
@@ -559,6 +560,11 @@
 			<var name="ol3"/>
 			<var name="ol4"/>
 			<var name="h_oml_initial"/>
+			<var name="soilcomp"/>
+			<var name="soilcl1"/>
+			<var name="soilcl2"/>
+			<var name="soilcl3"/>
+			<var name="soilcl4"/>
```

Comment:
In the current `develop` branch, other LSM fields (`isltyp`, `ivgtyp`, etc.) are in the `invariant` stream and therefore included in the `input` and `restart` streams. I'd propose we also move `soilcomp`, `soilcl1`, `soilcl2`, `soilcl3`, and `soilcl4` into the `invariant` stream as well.

Reply:
done. it turns out that I had also forgotten to add soilcl4 in the input stream. corrected this error as well.